pkgs/development/python-modules/dfdiskcache/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pandas
+, setuptools
+, simplesqlite
+, typing-extensions
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "df-diskcache";
+  version = "0.0.2";
+  format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/3842a48ca882941331f341c495fa5df1f96121f5